SU-SWS/stanford_profile
# Page Authoring: Banner Top

# READY FOR REVIEW 
https://github.com/SU-SWS/stanford_profile/pull/62/files#diff-c929b93fbed3d66a096df41d11b8f759R44

# Summary
_See ticket:  https://stanfordits.atlassian.net/browse/D8CORE-1112_

Here are some videos to demo how it works in Chrome:
https://nimb.ws/TpUr9g
https://nimb.ws/UqAZ1M

# Needed Review By (Date)
1/17/2020
 
# Urgency
- Somewhat Urgent

# Steps to Test

- Checkout this branch
- Create or edit a basic page
- Verify that it looks like what is expected in the ticket.

# Affected Projects or Products 
- D8Core (https://github.com/SU-SWS/acsf-cardinalsites)

# QA
## HTML Validation Results (https://validator.w3.org/)
_Click on `Message Filtering` and paste results or provide URL_

## Accessibility Testing 
### AMP Asset URL:
### Automated tool(s) used and results (ACCESS Assistant, WAVE, AXE): 
WAVE detects 
- 8 missing form labels,
- 2 contrast errors

### Manual testing performed:
 - None

## Cross-Browser testing:
What browsers have been tested?
- Chrome - Yes
- Firefox - yes
- Safari
- Internet Explorer
- Android (Edge)
- iPhone (Safari)

## Unit Testing


# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/D8CORE-941

## Related PRs


## More Information
The ticket didn't specify how to handle hover and focus. I added some borders on hover to make things stand out a bit.

## Folks to notify
`@sherakama` 
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
